zebra/if_netlink.c

```diff
@@ -672,7 +672,7 @@ static int netlink_interface(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 	if (vrf_is_backend_netns())
 		vrf_id = (vrf_id_t)ns_id;
 
-	vrf = vrf_lookup_by_id(vrf_id);
+	vrf = vrf_get(vrf_id, NULL);
```

Comment:
What's the sequence of events that leads us to getting interface events for a vrf before we have information about that vrf?

Reply:
I don't remember exactly. I think that while testing. I encountered this issue.
I propose to skip this commit for now.